src/main/java/lotto/controller/LottoMain.java

```diff
@@ -0,0 +1,24 @@
+package lotto.controller;
+
+import java.util.List;
+import lotto.domain.Lotto;
+import lotto.utils.TextManipulator;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoMain {
+
+        public static void main(String args[]) {
```

Comment:
자바에서 배열은 타입에 표기합니다
[컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.3-arrays) 확인 부탁드려요
```suggestion
        public static void main(String[] args) {
```

애플리케이션을 실행하면 정상적으로 동작하지 않네요 😢 
![image](https://github.com/next-step/java-lotto/assets/8064743/e80d7544-c48b-4021-a289-24b3ce4db970)

Reply:
어제.. 제가 분명 Main 실행 출력 제대로 되는 것까지 보고 MR 올렸었는데..  
backup용 branch랑 꼬인 채 MR 올린 것 같습니다.. 치명적인 실수를 했네요,,
정정하겠습니다 ㅠ-ㅠ